docs/userguide/dockervolumes.md

```diff
@@ -119,8 +124,8 @@ create it for you.
 > host-dependent, so a host directory specified in a `Dockerfile` probably
 > wouldn't work on all hosts.
 
-Docker defaults to a read-write volume but we can also mount a directory
-read-only.
+As described before, Docker defaults to a read-write volume but we can
```

Comment:
I think the "As described before," is not needed (Sure, it may be duplicated information but that doesn't hurt).